main.js

```diff
@@ -106,56 +113,62 @@ function toggleRulerCommand() {
 }
 
 app.dock.hide();
-app.on('ready', function() {
+app.on('ready', function () {
 	let trayIcon = new Tray(path.join(__dirname, 'src/assets/images/lrTemplate.png'));
 
 	var trayMenuTemplate = [
-			{
-					label: 'New ruler',
-					accelerator: 'Command+Ctrl+R',
-					click: createNewRuler
-			},
-			{
-					label: 'Toggle rulers',
-					accelerator: 'Command+Ctrl+T',
-					click: toggleRulerCommand,
-					enabled: false
-			},
-			{
-				type: 'separator'
-			},
-			{
-				label: 'Settings',
-				click: () => {
-					if (settingsWindow) {
-						return;
-					}
-
-					settingsWindow = new BrowserWindow({
-						width: 200,
-						height: 170,
-						alwaysOnTop: true
-					});
-
-					settingsWindow.loadURL(`file://${__dirname}/src/app/settings/settings.html`);
-
-					settingsWindow.on('closed', () => {
-						settingsWindow = null;
-					});
+		{
+			label: 'New Ruler',
+			accelerator: 'Command+Ctrl+R',
+			click: createNewRuler
+		},
+		{
+			label: 'Toggle Rulers',
+			accelerator: 'Command+Ctrl+T',
+			click: toggleRulerCommand,
+			enabled: false
+		},
+		{
+			label: 'Toggle Center Guides',
+			accelerator: 'Command+;',
+			click: toggleCenterGuidesCommand,
+			enabled: true
```

Comment:
false alarm. my original thinking is correct, it would require us to make a state machine and rebuild the menu based on the pub-sub of various app-level events. Its a cool idea, but the re-write would take us beyond the scope of this PR.